src/datachain/remote/studio.py

```diff
@@ -119,18 +119,27 @@ def _check_dependencies(self) -> None:
                 "\tpip install 'datachain[remote]'"
             ) from None
 
-    def _send_request_msgpack(self, route: str, data: dict[str, Any]) -> Response[Any]:
+    def _send_request_msgpack(
+        self, route: str, data: dict[str, Any], method: Optional[str] = "post"
+    ) -> Response[Any]:
         import msgpack
         import requests
 
-        response = requests.post(
-            f"{self.url}/{route}",
-            json={**data, "team_name": self.team},
+        kwargs = (
+            {"params": {**data, "team_name": self.team}}
+            if method == "get"
```

Comment:
Doesn't really matter, but it is more common to see HTTP methods in uppercase 👀
Also see [here](https://github.com/psf/requests/blob/23540c93cac97c763fe59e843a08fa2825aa80fd/src/requests/api.py#L17).
```suggestion
            if method.upper() == "GET"
```